internal/pkg/service/stream/plugin/collection.go

```diff
@@ -59,9 +95,54 @@ func (c *Collection) OnSourceSave(fn onSourceSaveFn) {
 	c.onSourceSave = append(c.onSourceSave, fn)
 }
 
+func (c *Collection) OnSourceEnabled(fn onSourceSaveFn) {
+	c.onSourceSave = append(c.onSourceSave, func(ctx context.Context, now time.Time, by definition.By, original, branch *definition.Source) error {
```

Comment:
`branch` name used as `*definition.Source`

Reply:
:+1: I will fix it

Reply:
https://github.com/keboola/keboola-as-code/pull/1792/commits/746f67df64188dc4de776baa4858614f378da6c1